chaos/navitia.py

```diff
@@ -112,17 +112,17 @@ def get_pt_object(self, uri, object_type, pt_objects=None):
 
         if response:
             json = response.json()
+            timeout = app.config['CACHE_CONFIGURATION'].get('NAVITIA_CACHE_TIMEOUT', 3600)
```

Comment:
```suggestion
            cache_timeout = app.config['CACHE_CONFIGURATION'].get('NAVITIA_CACHE_TIMEOUT', 3600)
```

as already used for Navitia timeout